sample/src/main/java/co/nimblehq/sample/ui/surveydetail/SurveyDetailModelFactory.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.sample.ui.surveydetail
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+/**
+ * ViewModel provider factory to instantiate LoginViewModel.
+ * Required given LoginViewModel has a non-empty constructor
+ */
+class SurveyDetailModelFactory : ViewModelProvider.Factory {
```

Comment:
We have to duplicate these steps multiple times. I'd recommend handling the ModelFactory by using Dagger or Hilt when we have more spare time.

https://github.com/nimblehq/survey-api-sdk-android/blob/e61fafb3f536d5531068e72d2dba27f39941f705/sample/src/main/java/co/nimblehq/sample/ui/login/LoginViewModelFactory.kt#L10-L19

https://github.com/nimblehq/survey-api-sdk-android/blob/e61fafb3f536d5531068e72d2dba27f39941f705/sample/src/main/java/co/nimblehq/sample/ui/surveylist/SurveyListModelFactory.kt#L10-L19

Reply:
@hoangnguyen92dn I think we will do it later, after complete the `Sdk`. Do it as an `refactor` or `improvement` in spare time.

Reply:
@doannimble @hoangnguyen92dn MVVM architecture is optional in sample app, leave it for later please 🙏

Reply:
@doannimble @luongvo I recommended for improvement when we have more spare time. Currently, we can leave it as it is for now.